here-naksha-cli/src/jvmMain/java/com/here/naksha/cli/copy/service/CopyService.java

```
@@ -31,12 +35,13 @@ public CopyService(
     @NotNull
     public CommandResult<CopyServiceSuccessResultPayload, CopyServiceException> copy(
             @NotNull CopyElement src,
-            @NotNull CopyElement target
+            @NotNull CopyElement target,
+            boolean autoCreateTarget
     ) {
         try {
             List<NakshaFeature> features = readFeaturesFromSrc(src);
```

let's `autoCreate` early on - before reading and writing - "fail fast" ;)

---

ok